### ISSUE TITLE
About localization

### ISSUE THREAD

**Issue by bfsujason** [notesjor/corpusexplorer2.0#12]
Is it possible to localize the interface of CorpusExplorer 2.0 into other languages?

For example, which file should I edit if I want to translate it into Chinese?

Thanks.

**Comment by notesjor** [comment]
Yes, this is expected to be possible this year. An English translation is available, but has not yet been implemented. Do you have experience with translations?

**Comment by bfsujason** [comment]
Thank you for your reply.

Yes, I have experience in English-Chinese translations in the field of corpus linguistics and computational linguistics. I think I'm able to handle the localization task if you could offer me the English version of CorpusExplorer 2.0.

Thanks.
